docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       - .:/code
     env_file:
       - ssh-keys.env
-    command: composer ci
+    command: sh -c 'composer install -n && composer ci'
```

Comment:
This could be a part of the Dockerfile

https://github.com/keboola/component-generator/blob/master/templates/php-component/Dockerfile#L20-L28

Reply:
yes, thanks, that's much better